nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -146,9 +146,10 @@ def run(self, args, conn):
 
             client = self._create_client(args, build_ctx, deploy_args)
 
-            app_root = get_simulator_app_root(args.workspace, client.client_name)
+            app_root = get_simulator_app_root(args.simulator_root, client.client_name)
             app_custom_folder = os.path.join(app_root, "custom")
-            sys.path.append(app_custom_folder)
+            if os.path.isdir(app_custom_folder) and app_custom_folder not in sys.path:
+                sys.path.append(app_custom_folder)
```

Comment:
Does simulator_runner run first or simulator_worker?
If simulator_runner run first we will remove the last item of sys.path?

Reply:
simulator_runner run first. It will add the server custom folder. It needs to be removed when running the simulator_worker